src/chi/components/alert/_alert-common.scss

```diff
@@ -43,45 +37,35 @@ $states: (
     border-width: 0.0625rem;
     box-sizing: border-box;
     color: $text-color;
+    display: flex;
     font-size: $text-small;
-    line-height: $line-height-smaller;
-    padding: 0.6875rem 0.9375rem;
+    line-height: $line-height-small;
+    padding: 0.5625rem 1rem;
     position: relative;
 
-    &::before {
-      content: attr(data-title);
-      display: block;
-      font-size: $text;
-      font-weight: 600;
-      text-transform: capitalize;
+    & i {
```

Comment:
@davidojedacl this property is giving 1 extra rem to the x button. Please make this property more specific to be attached to the icons, not to the button